pysb/tests/test_sensitivity_analysis.py

```diff
@@ -1,12 +1,13 @@
 from pysb.tools.sensitivity_analysis import \
-    InitialsSensitivity
+    PairwiseSensitivity, InitialsSensitivity
 from pysb.examples.tyson_oscillator import model
 import numpy as np
 import numpy.testing as npt
 import os
 from pysb.simulator.scipyode import ScipyOdeSimulator
 import tempfile
 import shutil
+from nose.tools import *
```

Comment:
Should be `from nose.tools import raises`. I know there are wildcard imports in other tests - we should replace those in the future too. PEP8 says to avoid them.